.github/workflows/macos.yml

```
@@ -39,6 +45,13 @@ jobs:
           brew update --preinstall
           brew bundle --file=contrib/Brewfile
 
+          OS_NAME=$([[ ${{ matrix.version }} -eq 13 ]] && echo "Ventura" || echo "Sonoma")
```

You might want to do something like this here:
```suggestion
case ${{ matrix.version }} in
  13)
    OS_NAME="Ventura"
    ;;
  14)
    OS_NAME="Sonoma"
    ;;
  *)
    echo "MacOS platform ${{ matrix.version }} is not supported by the pipeline!"
    exit 1
    ;;
esac
```

this way if we want to add another os we will not treat it as Sonoma but will die before trying to install the wrong version of the package

---

This is kind of a hardcoding now and it is intentional, as 
- both Github and the MacPorts side can change anytime
- this ensures we use the exactly here mentioned packages
It must be updated always manually if a new OS or package version is available.
There is a tiny chance we want to add a new OS version, as that would increase the build time again, instead, we will have to bump the OS versions parallel when needed

The `runs-on: macOS-${{ matrix.version }}` itself will already break the build if one of the currently used OS versions becomes unavailable anyway that will signal us to review/modify this part as needed (once a year probably when the new macOS version will be available on GitHub)